packages/gateway-repo/package.json

```diff
@@ -7,21 +7,25 @@
   "author": "Jacob Ebey",
   "license": "MIT",
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "deploy": "echo noop",
+    "setup:aws": "echo noop"
   },
   "dependencies": {
     "@apollo/federation": "^0.14.1",
     "@apollo/gateway": "^0.14.1",
     "@babel/polyfill": "^7.8.7",
     "apollo-server": "^2.12.0",
     "apollo-server-micro": "^2.12.0",
+    "aws-sdk": "^2.662.0",
+    "fs-extra": "^9.0.0",
     "node-fetch": "^2.6.0",
     "webpack-dependency-stats": "^0.0.3"
   },
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "babel-loader": "^8.1.0",
-    "webpack": "git://github.com/webpack/webpack.git#dev-1",
+    "webpack": "git://github.com/jacob-ebey/webpack.git#dev-1",
```

Comment:
If there is already a way to retrieve the remote entries's module chunk ID's that would be awesome. I couldn't find anything so I went ahead and forked webpack and added it to the ContainerEntryModule as "exposedModules" alongside of "get" and "override".